app/javascript/react/pages/NotFoundPage/NotFoundPage.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+const NotFoundContainer = styled.div`
```

Comment:
Can we approach it more universally, so it can be reused later easily if needed? I mean naming, what about just `Container` or `PageContainer`